docs/userGuide/syntax/modals.mbdf

```diff
@@ -6,25 +6,49 @@
 <span id="code">
 
 ```html
-More about <trigger for="modal:trigger_id">trigger</trigger>.
-<modal title="**Modal title** :rocket:" id="modal:trigger_id">
-    ...
+More about <trigger for="modal:loremipsum">trigger</trigger>.
+<modal title="**Modal title** :rocket:" id="modal:loremipsum">
+  Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
+  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
 </modal>
 <br>
-This is the same <trigger for="modal:trigger_id">trigger</trigger> as last one.
+This is the same <trigger for="modal:loremipsum">trigger</trigger> as last one.
+
+<trigger for="modal:centered">This is a trigger for a centered modal</trigger>.
+
+<modal add-class="text-center" id="modal:centered" center>
+  <div class="text-center" slot="modal-header">
```

Comment:
Let's revert back to not using slots in our example, to avoid the issue of disappearing `okay` button.